tests/roles/keystone_adoption/tasks/main.yaml

```diff
@@ -56,6 +56,12 @@
   retries: 60
   delay: 2
 
+- name: verify that OpenStackControlPlane setup is complete
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc wait --for=condition=Ready --timeout=1m OpenStackControlPlane openstack
```

Comment:
It won't hurt to add this, but if previous task pass this will always pass. Maybe the order of checks should be reverted. Check on OpenShift level first and then on Openstack level.

Just a side note: I was thinking at some point how it is possible that it is working as OpenStackControlPlane is not fully deployed yet returns condition=Ready.
I came to realization that is, because we are enable services one-by-one and by doing so this will return Ready state after all currently adopted services are deployed.

Reply:
Based on your earlier comment

> This should be after openstack endpoint list | grep keystone as in test, or test should be fixed to be consistent

I synced the docs to use the same order as tests.

Reply:
ack